Resources/Prototypes/Hydroponics/seeds.yml

```
@@ -357,6 +357,31 @@
       Max: 5
       PotencyDivisor: 5
 
+- type: seed
+  id: TeaPlant
```

```suggestion
  id: teaPlant
```
All the other seed prototypes use camel case, so let's keep it consistent.
You will have to adjust it where it is used as well.